pkg/synthesizer/speakers/piper.go

```diff
@@ -32,12 +34,12 @@ func NewPiperSpeaker(v voices.Voice, playbackSpeed float32) (Speaker, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to create speaker: %w", err)
 	}
-	return &piperSynth{tts: tts, playbackSpeed: playbackSpeed}, nil
+	return &piperSynth{tts: tts, playbackSpeed: playbackSpeed, playbackPause: playbackPause}, nil
 }
 
 // Say implements Speaker.Say
 func (s *piperSynth) Say(text string) ([]float32, error) {
-	synthesized, err := s.tts.Synthesize(text, piper.WithSpeed(s.playbackSpeed))
+	synthesized, err := s.tts.Synthesize(text, piper.WithSpeed(s.playbackSpeed), piper.WithPause(float32(s.playbackPause)))
```

Comment:
Casting a Duration to float uses the base time unit, which is nanoseconds rather than seconds. You'll want to use `s.playbackPause.Seconds()` instead of casting.

Reply:
thx. sadly I still need to convert to float32 because `s.playbackPause.Seconds()` returns float64